crates/ide-assists/src/handlers/generate_delegate_methods.rs

```diff
@@ -153,12 +151,12 @@ pub(crate) fn generate_delegate_methods(acc: &mut Assists, ctx: &AssistContext<'
                             Some(cap) => {
                                 let offset = strukt.syntax().text_range().end();
                                 let snippet = render_snippet(cap, impl_def.syntax(), cursor);
-                                let snippet = format!("\n\n{snippet}");
+                                let snippet = format!("\n\n{}", snippet);
```

Comment:
Unnecessary change

Reply:
I made this change cause I got a conflict from another PR, which did the same thing all over different assists